<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>photogrammetryClass</class>
 <widget class="QWidget" name="photogrammetryClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>影像配准</string>
  </property>
  <widget class="QLabel" name="LeftImg">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>171</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="RightImg">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>181</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Caculation">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Load_Right_Button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>导入右影像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Load_Left_Button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>导入左影像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>292</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请输入横坐标偏移量，建议值130,或1850</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="GetDelta">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="photogrammetry.qrc"/>
 </resources>
 <connections/>
</ui>
